admin/language/tr-tr/extension/payment/iyzico_checkout_form.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+// Heading
+$_['heading_title'] = 'Iyzico Ödeme Formu';
+
+// Text 
+$_['text_link'] = 'iyzico pazar yeri sistemine kayıt olmak için';
+$_['text_payment'] = 'ödeme';
+$_['text_success'] = 'Ayarlar güncellendi';
+$_['text_iyzico_checkout_form'] = '<a href="http://www.kahvedigital.com/" target="_blank"><img src="view/image/payment/iyzico_checkout_form.png" alt="iyzico" title="iyzico"/></a>';
+$_['text_iyzico_checkout_form_info'] = "Apı Id ve Secret Key anahtarlarınızı <a href='https://merchant.iyzipay.com/settings' target='_blank'>https://merchant.iyzipay.com/settings</a> 'dan alabilirsiniz.";
+// Entry
+$_['entry_api_id_live'] = 'Live Api Id';
+$_['entry_secret_key_live'] = 'Live Secret Key';
+$_['entry_order_status'] = 'Sipariş Durumu';
+$_['entry_status'] = 'Durumu';
+$_['entry_sort_order'] = 'Sıralama Türü';
+$_['entry_error_version'] = 'Yeni bir versiyon mevcut güncellemek için tıklayınız.';
+$_['entry_error_version_updated'] = 'Güncelleme Hatası! güncellemek için iyzico entegrasyon sayfasına giderek indirebilirsiniz. <a href="https://www.iyzico.com/entegrasyon/hazir-altyapi/opencart/" target="_blank">Entegrasyon sayfası</a> ';
```

Comment:
"Güncelleme Hatası! Manuel olarak güncellemek için <a href="https://www.iyzico.com/entegrasyon/hazir-altyapi/opencart/" target="_blank">tıklayın</a>."